tests/test_multitiff_multipage_imaging_extractor.py

```diff
@@ -0,0 +1,34 @@
+from roiextractors import MultiTiffMultiPageImagingExtractor
+
+from tests.setup_paths import OPHYS_DATA_PATH
+
+
+def test_init_multitiff_multipage_imaging_extractor():
+    extractor = MultiTiffMultiPageImagingExtractor(
+        folder_path=OPHYS_DATA_PATH / "imaging_datasets" / "Tif" / "splits",
+        pattern="split_{split:d}.tif",
+        sampling_frequency=1.0,
+    )
+
+    assert extractor.get_num_channels() == 1
+    assert extractor.get_num_frames() == 2000
+    assert extractor.get_sampling_frequency() == 1.0
+    assert extractor.get_channel_names() is None
+    assert extractor.get_dtype() == "uint16"
+    assert extractor.get_image_size() == (60, 80)
+    assert extractor.get_video().shape == (2000, 60, 80)
+    assert list(extractor.tif_paths.keys()) == [
+        str(OPHYS_DATA_PATH / "imaging_datasets" / "Tif" / "splits" / x)
+        for x in (
+            "split_1.tif",
+            "split_2.tif",
+            "split_3.tif",
```

Comment:
Some extra tests for this, at a minimum, should include cases of (a) multiple cameras/microscopes (the specifier for each device being something in the file name), and (b) multiple optic channels for a single one of those devices (the channel name/ID also being a part of the filename)

Not that I'm not saying the `TiffFolderImagingExtractor` should by default support multiple optic channels, or auto-detection of channel IDs from this pattern (that is a more complicated feature that could be done in a follow-up, or not at all); I just want to make sure that a single instance made from a pattern specification that restricts to only one channel from a folder that contains several correctly identifies list of files only for that channel